client/js/components/admin/instances/new_instance_modal.jsx

```diff
@@ -1,7 +1,8 @@
 import React              from 'react';
 import ReactModal         from 'react-modal';
 import ReactSelect        from 'react-select';
-import NewDomainModal     from './new_domain_modal';
+import _                  from 'lodash';
+import DomainModal        from './new_domain_modal';
```

Comment:
So apparently you can't name the import the same name as a function in the imported file if it isn't the default export. Complies with https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-as-default.md